### PR TITLE
Tasks1-6

### PR DESCRIPTION
**Pull request by nazar-ginailo**
Tasks 1-6 without 5

### CODE REVIEW

File: training/1_populate_table/script.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        age: 23,
+        isWorkExperience: true
+    },
+];
+
+function arrOfObjectsToTable(obj) {
+    let table = '<table>';
+    table += '<tr>';
+    // let keys = Object.keys(students[0]);
+    for( let key in students[0] ) {
```
Comment (ArtemLunt) [inline training/1_populate_table/script.js]: IMO it's not really clear code.
We can use `Object#entries` method to walk through the object's keys and values in the same time.

File: tasks/1_golf-score/golf-score.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  GO_HOME: `Go Home`,
+};
+
+function golfScore(par, strokes) {
+  if (strokes === 1) {
+    return GOLF_SCORES.HOLE_IN_ONE;
+  }
+  switch (strokes - par) {
+    case -2:
+    case -3:
```
Comment (Tsvetkovv) [inline tasks/1_golf-score/golf-score.js]: There is a condition in the task description
> <= par - 2 | “Eagle”

if stroke = 2 and par = 6, this function returns `GOLF_SCORES.GO_HOME` instead of `GOLF_SCORES.EAGLE`

File: tasks/5_profile-lookup/profile-lookup.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    'likes': ['Javascript', 'Gaming', 'Foxes'],
+  },
+];
+
+export function lookupProfile(firstName, prop) {
+  const contact = contacts.find((cont) => cont.firstName === firstName);
+  if (!contact) {
+    return 'No such contact';
+  }
+  if (contact.hasOwnProperty(prop)) {
```
Comment (Tsvetkovv) [inline tasks/5_profile-lookup/profile-lookup.js]: [in operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in)

File: tasks/6_reverse-string/reverse-string.spec.js [hunk 4]
```diff
@@ -15,9 +15,4 @@ xdescribe('reverse-string', () => {
 
     test('reverseString("Greetings from Earth") should return "htraE morf sgniteerG"', () => {
         expect(reverseString('Greetings from Earth')).toBe('htraE morf sgniteerG');
-    });
-
-    test('reverseString("foo bañana mañana") should return "anañam anañb oof"', () => {
```
Comment (Tsvetkovv) [inline tasks/6_reverse-string/reverse-string.spec.js]: why did you remove this test case?

Comment (Tsvetkovv) [reply tasks/6_reverse-string/reverse-string.spec.js]: please merge master to this branch and fix the tests if required
